tests/test_hf_model.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+import torch
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+prompts = ["你好", "what's your name"]
+
+
+def assert_model(response):
+    assert len(response) != 0
+    assert "user" not in response
+    assert "bot" not in response
```

Comment:
We may remove it since it's not stable sometimes.

Reply:
Get!